src/sentry/ratelimits/utils.py

```diff
@@ -134,9 +139,10 @@ def get_organization_id_from_token(token_id: int) -> int | None:
     installations = app_service.get_many(
         filter={
             "status": SentryAppInstallationStatus.INSTALLED,
-            "api_token_id": token_id,
+            "api_installation_token_id": token_id,
```

Comment:
We can't start using new RPC methods and parameters in the same pull request that the method or parameter is defined in. The reason for this is deploys don't go out to all regions and control at the same time. Regions deploy first, and then control. With the current changes you would be sending an unsupported filter condition to control for the duration of the deploy (20 min or so).

Instead we need to deploy the new parameter support first, and then deploy changes that use the new filter option.

Reply:
Just merged https://github.com/getsentry/sentry/pull/67159, waiting for it to deploy

Reply:
deploy went out, just have to merge this in now